src/invidious/helpers/utils.cr

```diff
@@ -18,23 +18,39 @@ def elapsed_text(elapsed)
   "#{(millis * 1000).round(2)}µs"
 end
 
-def decode_length_seconds(string)
-  length_seconds = string.gsub(/[^0-9:]/, "")
-  return 0_i32 if length_seconds.empty?
+module TimeSpanConverter
```

Comment:
Please put that module in `config.cr`, with the other converters.

Reply:
I'm moving this right now

Reply:
By the way the other conversion modules are in preferences, I have to put it in `user/preferences.cr` in this case even if it is not used for the config or I put it in `config.cr`, where there is currently no conversion module ?

Reply:
Oh, right, I haven't move those yet! Yeah, put them in `config.cr`.

Reply:
Ok !